containers/filters/editor/Sieve.js

```diff
@@ -63,7 +67,7 @@ function FilterEditorSieve({ filter, onChangeBeforeLint = noop, onChange = noop
     return (
         <CodeMirror
             className="bordered-container"
-            value={filter.Sieve}
+            value={value}
```

Comment:
And the `onChange` prop doesn't need to interact with the `setValue` function?

Reply:
Good remark which made me read a bit about [uncontrolled](https://reactjs.org/docs/uncontrolled-components.html) and [controlled](https://reactjs.org/docs/forms.html#controlled-components) components.

As we use the uncontrol version of codemirror, this seems to work well because the value is only used at initialization. In controlled version, it would indeed fail.

But alternatively, I could also not messing with the value. Considering my tests, any React render will fix the problem. I can create a dummy state to artificially trigger a render. But as it looked strange, I prefered link this to an actual value.